opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -99,6 +99,9 @@ namespace Opm {
         using SourceDataSpan = typename
             PAvgDynamicSourceData<Scalar>::template SourceDataSpan<Scalar>;
 
+        using SourceDataSpan =
```

Comment:
This duplicates the above, maybe a rebasing error.

Reply:
hopefully fixed